src/panoptes/pocs/focuser/birger.py

```diff
@@ -64,57 +63,18 @@ def __init__(self,
                  initial_position=None,
                  dev_node_pattern='/dev/tty.USA49*.?',
                  max_command_retries=5,
+                 serial_number_pattern=serial_number_pattern,
```

Comment:
Similar to the astromechanics, the `dev_node_pattern` should default to `None` and these parameters probably don't need to be specified.